src/pyasm/search/database_impl.py

```diff
@@ -3364,30 +3364,36 @@ def get_serial(self, length=4, not_null=False):
         parts.append("serial")
         return " ".join(parts)
 
-    def get_text(self, not_null=False):
+    def get_text(self, default=None, not_null=False):
```

Comment:
Can you please add the new parameter as the last argument so that it backwards compatible?

Reply:
Done. Thanks for pointing it out